src/lib/Errors.js

```diff
@@ -81,6 +81,12 @@ Errors.RequestExpired = class extends Errors.KeyguardError {
     }
 };
 
+Errors.ResetPassword = class extends Errors.KeyguardError {
```

Comment:
To keep in the spirit of the other GOTO error.

I would also define this error _after_ the other GOTO one (that's where I first looked for it and R comes _after_ C in the alphabet).

```suggestion
Errors.GoToResetPassword = class extends Errors.KeyguardError {
```